strongbox-commons/src/main/java/org/carlspring/strongbox/config/PropertiesPathResolverConfig.java

```diff
@@ -0,0 +1,106 @@
+package org.carlspring.strongbox.config;
+
+import java.nio.file.Paths;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.util.StringUtils;
+
+/**
+ * Configures a {@link PropertiesPathResolver} under the name "propertiesPathBean" - it can then be used in SpEL
+ * expressions to resolve properties lookup path's such as {@code @Value("#{propertiesPathBean.resolve('customPropName','defaultPropPath')}")}.
+ *
+ * @author cbono
+ */
+@Configuration
+class PropertiesPathResolverConfig
+{
+    @Bean(name = "propertiesPathBean")
+    PropertiesPathResolver propertiesPathResolver(Environment environment)
```

Comment:
`propertiesPathResolver` seems better name.

Reply:
I agree.

Reply:
Done. Nws. I should have named it that from the very beginning anyways. Try to be clever and see where it gets ya! LOL.